Source/items.cpp

```diff
@@ -1930,7 +1930,7 @@ _item_indexes RndPremiumItem(const Player &player, int minlvl, int maxlvl)
 	return RndVendorItem<PremiumItemOk>(player, minlvl, maxlvl);
 }
 
-void SpawnOnePremium(Item &premiumItem, int plvl, const Player &player)
+void SpawnOnePremium(Item &PremiumItems, int plvl, const Player &player)

```

Comment:
This is now shadowing the global, and shouldn't be renamed in the first place.

Reply:
ah crap